src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue

```diff
@@ -1,54 +1,68 @@
 <template>
   <div>
-    <b-row class="pb-5">
-      <b-col xs="12" sm="6" class="rounded-left first-box">
-        <!-- <ExtraWorkFeedback :data="data" :variant="'#9964e3 !important'"/> -->
-        <ExtraWorkFeedback :data="performanceData" :variant="'#9964e3 !important'"/>
-      </b-col>
-      <b-col xs="12" sm="6" class="rounded-right">
-        <b-alert
-          show
-          class="Alert-class"
-        >Info: Overall rating/ comments mandatory for managers based on weekly report</b-alert>
-        <div class="mb-3">
-          <h6 class="rating-header mb-2">Overall Rating</h6>
-          <div class="border-top"></div>
-          <div class="mt-2 font-weight">Based on Weekly Review</div>
-        </div>
-        <starRating
-          class="border-bottom"
-          :displayStar="5"
-          :ratedStar="ratedStarWeekly"
-          :starSize="starSize"
-          @starRatingSelected="submitStarRateWeekly"
-        />
-        <div
-          class="mt-2 font-weight"
-        >Difficulty level of Project (if project work you did was difficult/required more effort)</div>
-        <starRating
-          class="border-bottom"
-          :displayStar="5"
-          :ratedStar="ratedStarDifficulty"
-          :starSize="starSize"
-          @starRatingSelected="submitStarRateDifficulty"
-        />
-        <div sm="6">
-          <h6 class="text-inverse">Comments</h6>
-        </div>
-        <b-form>
-          <b-form-textarea
-            :rows="3"
-            v-model="text"
-            id="default-textarea"
-            placeholder="Performance or general comments (if any)..."
-          />
-          <b-button
-            class="btn btn-default btn-lg mb-xs bg-primary text-white mt-4"
-            @click="submit"
-          >Submit</b-button>
-        </b-form>
-      </b-col>
-    </b-row>
+    <span v-if="activeEmployee">
+      <span v-for="(reviews,index) in activeEmployee.is_reviewed" :key="index">
+        <span v-if="userProfile._id == reviews._id">
+          <b-row class="pb-5">
+            <b-col xs="12" sm="6" class="rounded-left first-box">
+              <!-- <ExtraWorkFeedback :data="data" :variant="'#9964e3 !important'"/> -->
+              <ExtraWorkFeedback :data="activeEmployee" :variant="'#9964e3 !important'"/>
+            </b-col>
+            <b-col xs="12" sm="6" class="rounded-right">
+              <b-alert
+                show
+                class="Alert-class"
+              >Info: Overall rating/ comments mandatory for managers based on weekly report</b-alert>
+              <div class="mb-3">
+                <h6 class="rating-header mb-2">Overall Rating</h6>
+                <div class="border-top"></div>
+                <div class="mt-2 font-weight">Based on Weekly Review</div>
+              </div>
+              <span :disabled="reviews.reviewed == true ? true : false">
```

Comment:
again remove this ternary operator as well

Reply:
removed ternary operator